modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -216,12 +217,37 @@ export default class MVTLayer extends TileLayer {
     return renderedFeatures;
   }
 
+  getVisibleTiles(tileCoords = 'local') {
+    const {selectedTiles} = this.state.tileset;
+    const {viewport} = this.context;
+
+    if (conversionToWGS84isRequired(tileCoords, viewport.resolution)) {
+      this._convertTilesToWGS84(selectedTiles, viewport);
+    }
+
+    return selectedTiles;
+  }
+
+  _convertTilesToWGS84(tiles, viewport) {
+    const {tileIdsCache} = this.state;
+
+    tiles.forEach(tile => {
+      const tileId = `${tile.x},${tile.y},${tile.z}`;
+
+      if (!tileIdsCache.has(tileId)) {
+        tileIdsCache.add(tileId);
```

Comment:
What is the purpose of this? Just check if `transformToWorld` is assigned?

Reply:
Indeed, checking if calculation is done for a certain tile, avoiding unnecessary calculations. We can even 
simplify it:

```javascript
if (!tile.transformToWorld ) {
  tile.transformToWorld = () => updateTileWithWGS84Coords(tile, tile.content, viewport);
}
```

Reply:
> avoiding unnecessary calculations

Assigning `transformToWorld` is not doing any actual calculation.